src/popup/components/Modals/QrCodeReader.vue

```diff
@@ -179,7 +210,13 @@ export default {
         window.plugins.webviewcolor.change('#141414');
         this.setQrScanner(false);
         window.QRScanner.destroy();
-      } else this.browserReader.reset();
+      } else {
+        try {
```

Comment:
we should do somethink with catch, or.. if is not nessesary remove the try catch

@peronczyk @pstachel what do you think?